tests/test_wave_05.py

```diff
@@ -80,34 +83,45 @@ def test_create_goal(client):
     }
 
 
-@pytest.mark.skip(reason="test to be completed by student")
+# @pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    raise Exception("Complete test")
+    #raise Exception("Complete test")
     # Act
     # ---- Complete Act Here ----
-
+    response = client.put("/goals/1", json={
+        "title": "Updated goal Title"})
+    response_body = response.get_json()
     # Assert
+    assert response.status_code == 200
+    assert "goal" in response_body
+    assert response_body == {"goal": {
+        "id": 1,
+        "title": "Updated goal Title"}}
+    goal = Goal.query.get(1)
+    assert goal.title == "Updated goal Title"
```

Comment:
👍  Good work query the goal table again to confirm that the goal title matched what was expected!